### PR TITLE
Onboarding mandatory analytics, save seed subtitle, social tooltips

### PR DESCRIPTION
**Pull request by dasilvarosa**
Wording

Included wording for all three shared data types (sentry, analytics, and mandatory contextual data)

Onboarding

### CODE REVIEW

File: static/i18n/en/onboarding.yml [hunk 1]
```diff
@@ -56,16 +56,18 @@ selectPIN:
       nano:
         step1: Connect the Ledger Nano S to your computer.
         step2: Press both buttons simultaneously as instructed on the screen.
-        step3: Press the left button to cancel <1><0>Initialize as new device?</0></1> Press the right button to select <3><0>Restore configuration?</0></3>
-        step4: 'Choose a PIN code between 4 and 8 digits long followed by the checkmark (✓).'
+        step3: 'Press the left button to cancel <1><0>Initialize as new device?</0></1>. Then press the right button to select <3><0>Restore configuration?</0></3>'
+        step4: 'Choose a PIN code between 4 and 8 digits , followed by the checkmark (✓).'
```
Comment (gre) [inline static/i18n/en/onboarding.yml]: no space before comma

File: static/i18n/en/onboarding.yml [hunk 2]
```diff
@@ -44,7 +44,7 @@ selectPIN:
       nano:
         step1: Connect the Ledger Nano S to your computer.
         step2: Press both buttons simultaneously as instructed on the screen.
-        step3: Press the right button to select <1><0>Configure as new device?</0></1>
+        step3: 'Press the right button to select <1><0>Configure as new device?</0></1>.'
```
Comment (gre) [inline static/i18n/en/onboarding.yml]: weird point after a `?`  if i read correctly?

File: static/i18n/en/onboarding.yml [hunk 3]
```diff
@@ -108,11 +110,11 @@ genuineCheck:
     contactSupport: Contact us
   errorPage:
     ledgerNano:
-      title: Oops, your device does not seem genuine...
+      title: Oops, something went wrong...
         #PIN: Didn't choose your own PIN code?
         #recoveryPhrase: Didn't save your own recovery phrase?
         #Genuine: Oops, your device does not seem genuine...
-      desc: Your device did not pass the authenticity test required to connect to Ledger’s secure server. Please contact Ledger Support to get assistance.
+      desc: Go back to the security checklist or request Ledger Support assistance
```
Comment (gre) [inline static/i18n/en/onboarding.yml]: 👌

File: src/components/DeviceConnect/index.js [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+              <TranslatedError error={error} />
+            </Box>
+          </Info>
+        ) : null}
+      </Box>
+    )
+  }
+}
+
+export default translate()(DeviceConnect)
```
Comment (meriadec) [inline src/components/DeviceConnect/index.js]: wtf

**Comment by gre** [comment]
you need to `git rm ` this file, it's gone now

**Pull request merged**